liiatools/cin_census_pipeline/stream_pipeline.py

```diff
@@ -14,12 +15,13 @@
 from . import stream_filters as filters
 
 
-def task_cleanfile(src_file: FileLocator, schema: (XMLSchema, Path), logger: Optional[logging.Logger]=None) -> ProcessResult:
+def task_cleanfile(src_file: FileLocator, schema: (XMLSchema, Path), output_config: PipelineConfig, logger: Optional[logging.Logger]=None) -> ProcessResult:
```

Comment:
I would probably just call this argument pipeline_config to stay consistent across the codebase

Reply:
Alright, though I don't think it's a very descriptive name for the purpose